src/infrastructure/content/NetworkMessageHandler.java

```diff
@@ -0,0 +1,79 @@
+package infrastructure.content;
+
+import networking.INotificationHandler;
+import org.json.JSONObject;
+import java.util.HashMap;
+
+/*
+ * NetworkMessageHandler class implements the INotificationHandler interface.
+ * this includes one method i.e @onMessageReceived()
+ * this class is used to notify and to send data to the UI module 
+ * as new user joined or as a user exit or as message sent by a user
+ * 
+ * @author Talha Yaseen (111701019)
+ */
+public class NetworkMessageHandler implements INotificationHandler {
+	
+	/*
+	 * Creating instance of ContentCommunicator class to access map handler and image map
+	 */
+	private ContentCommunicator cc = new ContentCommunicator();
+	
+	/*
+	 * Accessing mapHandler hashmap from ContentCommunicator class to use it for sending message to UI
+	 */
+	private HashMap<String, IContentNotificationHandler> mapHandler = cc.getMapHandler();
+	
+	/*
+	 * Accessing mapImage hashmap from ContentCommunicator class to access image corresponding to username
```

Comment:
Follow [this](https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line) for multi line javadoc comments. Change it in other places too.